exporters/ibmmq/ibmmq-config.yml.sample

```diff
@@ -32,4 +32,20 @@ integrations:
         #       - "go_"
         #       - "process_"
 
+        # The following parameters will be passed as environment variables directly to the IBM mq exporter in order to set up advanced connection setups such as TLS and mTLS.
 
+        # MQSSLKEYR specifies the location of the key repository that holds the digital certificate belonging to the user or to the server.
+        # https://www.ibm.com/docs/en/ibm-mq/9.1?topic=variables-mqsslkeyr
+        # mqsslkeyr: /key
+
+        # Path to the ccdt file. Two different environment variable will be set automatically: MQCCDTURL and IBMMQ_CONNECTION_CCDTURL.
+        # MQCCDTURL is needed by the IBMMQ client, IBMMQ_CONNECTION_CCDTURL by the exporter.
+        # ccdt_url: file:///ccdt.json
+
+        # IBMMQ Client needs permissions to write in a directory specified in the HOME environment variable
+        # https://www.ibm.com/support/pages/apar/IT32860
+        # home: /tmp
```

Comment:
should we change this variable name by adding a prefix?

Reply:
No. It was mandatory that wa: https://www.ibm.com/docs/en/ibm-mq/9.1?topic=reference-environment-variables